config/spryk/templates/composer.json.twig

```diff
@@ -3,7 +3,12 @@
   "description": "{{ module }} module",
   "license": "proprietary",
   "require": {
-    "php": ">=7.1"
+    "php": ">=7.1",
+    "spryker/kernel": "*"
```

Comment:
This might get flagged by tooling.
We cant use a real value?

Reply:
We can't as this is a moving target and we can't know which one is the correct version at this time.

Reply:
Do you expect Kernel major to go up this often?
With this people will always have to adjust their modules after running spryks.. That is far less ideal than us maintaining the major here (which we can even automate using a post-hook for Kernel majoring :smile: )